runatlantis.io/docs/server-configuration.md

```diff
@@ -927,6 +927,13 @@ and set `--autoplan-modules` to `false`.
   environment where releases.hashicorp.com is not available. Directory structure of the custom
   endpoint should match that of releases.hashicorp.com.
 
+### `--tf-disable-downloads`
+  ```bash
+  atlantis server --tf-disable-downloads
```

Comment:
Please add the common environment variable documentation that is shown in other command docs below and above

Reply:
Addressed in https://github.com/runatlantis/atlantis/pull/2843/commits/6ae09a25fd15c4e604fb4467f76dd91b10a9f91f